test/generators/database_cleaner/active_record/rspec/active_record_install_generator_test.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'generators/boring/database_cleaner/active_record/rspec/install/install_generator'
+
+class DatabaseCleanerActiveRecordInstallGeneratorTest < Rails::Generators::TestCase
+  tests Boring::DatabaseCleaner::ActiveRecord::Rspec::InstallGenerator
+  setup :build_app
+  teardown :teardown_app
+
+  include GeneratorHelper
+  include ActiveSupport::Testing::Isolation
+
+  def destination_root
+    app_path
+  end
+
+  def test_should_exit_if_rspec_is_not_installed
+    assert_raises SystemExit do
+      quietly { generator.verify_presence_of_rspec_gem }
+    end
+  end
+
+  def test_should_exit_if_rails_helper_is_not_present
+    assert_raises SystemExit do
+      quietly { generator.verify_presence_of_rails_helper }
+    end
+  end
+
+  def test_should_add_database_cleaner_active_record_gem
+    Dir.chdir(app_path) do
+      install_rspec
+
+      quietly { run_generator }
+
+      assert_gem 'database_cleaner-active_record'
+    end
+  end
+
+  def test_should_configure_database_cleaner_with_rspec
+    database_cleaner_template_content = read_database_cleaner_template
+    Dir.chdir(app_path) do
+      install_rspec
+
+      quietly { run_generator }
+
+      database_cleaner_content = File.read('spec/support/database_cleaner.rb')
+      assert_equal database_cleaner_template_content, database_cleaner_content
```

Comment:
I think this test can be pretty pointless because if generator ran successfully is there a chance that file is not created from the template? If we can instead write a test to check that test fails when content is not similar and change the content likewise then maybe that would make sense.

Reply:
is not it exactly doing this "test to check that test fails when content is not similar" as template will render different config based on flag passed